vizro-ai/src/vizro_ai/plot/components/dataframe_craft.py

```diff
@@ -43,8 +43,7 @@ class DataFrameCraft(BaseModel):
 2. Do not create any new DataFrames; work only with df.
 3. Always make a hard copy of the DataFrame before manipulating it. Important: Do not modify the original DataFrame.
 4. Ensure that any aggregated columns are named appropriately and re-indexed if necessary.
-5. If a visualization is implied by the user's query, only write the necessary DataFrame manipulation
-code for that visualization.
+5. Focus on providing operations related to aggregation, filtering, sorting, grouping, and other transformations.
```

Comment:
This is to avoid a common error found in integration test: ""error: matplotlib is required for plotting when the default backend "matplotlib" is selected.""